.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,20 @@
+### Summary
+
+<!-- _*(Concludes of related issues, a description or list of the changes and the motivation behind them)*_ -->
```

Comment:
```suggestion
<!-- _*(Consists of related issues, a description or list of the changes and the motivation behind them)*_ -->
```

Is Concludes correct in this context? If it is then we might use some more common word to be clearer anyway, or reword it and remove the prefix entirely so it reads "Related issues, a description or list of the changes and the motivation behind them".

btw. what does `_*` & `*_` change there?

Reply:
Nothing, I forgot it from before.